### ISSUE TITLE
Bad code generation after PR #528

### ISSUE THREAD

**Issue by andradei** [volatiletech/sqlboiler#544]
If you're having a generation problem please answer these questions before submitting your issue. Thanks!

### What version of SQLBoiler are you using (`sqlboiler --version`)?
SQLBoiler v3.4.0

This happens at **build time** because the code generation outputted undefined variables.

After #528 got merged, I'm still getting code generated with `{{$alias.DownSingular}}Columns` instead of the new `{{$alias.DownSingular}}AllColumns` (_All_ is missing in my generated models). In the generated code, the later wasn't defined but the latter was, so the generated code should... generate `{{$alias.DownSingular}}Columns`anymore. I don't know where to look to try to fix this with another PR, but I'm almost positive PR #528 caused this in v3.4.0.

This causes an `undefined: {structfieldname}Columns` error.

---

The issue seems to be _only_ in the models `Upsert` methods.

**Comment by andradei** [comment]
I made sure to clean the build cache and reinstall sqlboiler to make sure to cache or old package versions were somehow causing the problem.

**Comment by aarondl** [comment]
Have you also been sure to reinstall the drivers?

**Comment by andradei** [comment]
You're right. `go get -u -v github.com/volatiletech/sqlboiler/drivers/sqlboiler-psql` is what I was missing.

I forget both sqlboiler and the driver must be updated and got confused thinking `github.com/volatiletech/sqlboiler/drivers/sqlboiler-psql` was included as a default driver.

Sorry for the false positive.
